docusaurus/docs/packages/pkg/client/pkg.md

```diff
@@ -113,21 +112,278 @@ txctx -."tx query".-> chain
 go get github.com/pokt-network/poktroll/pkg/client
 ```
 
-## Usage
+## Usage (Supplier Client)
+
+### Basic Example (CLI command)
+
+```go
+package main
+
+import (
+  "context"
+  "log"
+  "net/url"
+
+  sdkflags "github.com/cosmos/cosmos-sdk/client/flags"
+  "github.com/spf13/cobra"
+
+  "github.com/pokt-network/poktroll/pkg/client/supplier"
+  "github.com/pokt-network/poktroll/pkg/deps/config"
+  sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+)
+
+var cmd = &cobra.Command{RunE: runCommand}
+
+func main() {
+  if err := cmd.ExecuteContext(context.Background()); err != nil {
+    log.Fatal(err)
+  }
+}
+
+func init() {
+  // IMPORTANT!: In order to take advantage of cosmos-sdk helpers which take
+  // the cobra command object  as an input, some flags MUST be registered and
+  // populated as expected by such helpers.
+  // See:
+  //   - https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.50.3/client#GetClientContextFromCmd
+  //   - https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.50.3/client/tx#NewFactoryCLI
+  cmd.Flags().String(sdkflags.FlagNode, "", "Register the default Cosmos node flag, which is needed to initialize the Cosmos query and tx contexts correctly.")
+  cmd.Flags().String(sdkflags.FlagGRPC, "", "Register the default Cosmos node grpc flag, which is needed to initialize the Cosmos query context with grpc correctly.")
+}
+
+func runCommand(cmd *cobra.Command, args []string) error {
+  // NOTE: interstitial `err` returns are ignored for brevity.
```

Comment:
`interstitial` is something I had to google and still find a little confusing - in an example try to keep it a little "dumber". What I _think_ this means is essentially:

```suggestion
  // NOTE: `err` returns are ignored for brevity.
```